.github/workflows/test.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           - python: '3.10'
             django: 'git+https://github.com/django/django.git@main#egg=Django'
```

Comment:
Should we also start testing for `@stable/4.1.x` (or maybe `4.1a1`)?

Reply:
Yep, we should replace the stable/4.0.x test above with stable/4.1.x, so that we catch any breaking changes in 4.1 between now and final release 👍